cmd/airflow.go

```diff
@@ -354,27 +354,27 @@ func newAirflowObjectRootCmd() *cobra.Command {
 func newObjectImportCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "import",
-		Short: "Create and update local Airflow objects from an Airflow Settings file",
-		Long:  "This command will create all connections, variables, and pools in an Airflow Settings file locally. Airflow must be running locally for this command to work",
+		Short: "Create and update local Airflow objects from a local YAML file",
+		Long:  "This command creates all connections, variables, and pools from a YAML configuration file in your local Airflow environment. Airflow must be running locally for this command to work",
 		// ignore PersistentPreRunE of root command
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
 			return nil
 		},
 		PreRunE: utils.EnsureProjectDir,
 		RunE:    airflowSettingsImport,
 	}
-	cmd.Flags().BoolVarP(&connections, "connections", "c", false, "Import connections from an Airflow Settings File")
-	cmd.Flags().BoolVarP(&variables, "variables", "v", false, "Import variables from an Airflow Settings File")
-	cmd.Flags().BoolVarP(&pools, "pools", "p", false, "Import pools from an Airflow Settings File")
-	cmd.Flags().StringVarP(&settingsFile, "settings-file", "s", "airflow_settings.yaml", "Settings or env file to export objects to")
+	cmd.Flags().BoolVarP(&connections, "connections", "c", false, "Import connections from a settings file")
+	cmd.Flags().BoolVarP(&variables, "variables", "v", false, "Import variables from a settings file")
+	cmd.Flags().BoolVarP(&pools, "pools", "p", false, "Import pools from a settings File")
+	cmd.Flags().StringVarP(&settingsFile, "settings-file", "s", "airflow_settings.yaml", "The YAML settings file from which to import Airflow objects. Default is 'airflow_settings.yaml'")
 	return cmd
 }
 
 func newObjectExportCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "export",
-		Short: "Export all Airflow objects to an airflow settings or env file. Does not overwrite objects that already exist in the file",
-		Long:  "This command will export all Airflow objects to an airflow settings or env file(--env-export). Objects already in the file will not be over written. Airflow must be running locally for this command to work",
+		Short: "Export all Airflow objects to a YAML or env file. Does not overwrite objects that already exist in the destination file",
```

Comment:
```suggestion
		Short: "Export all Airflow objects to a settings YAML or env file.",
```

Reply:
it doesn't overwrite the env file but it does overwrite the yaml file since it is a config. Configs are treated differently by the golang library